### DISCUSSION TITLE
Monorepo support

### DISCUSSION THREAD

**Discussion by MichalKlinka** [alirezanet/Husky.Net#83] [Q&A] [answered]
Hi, 
what is the recommended way of using this tool in monorepo? For example, if I have two solutions, and i want to use dotnet format as a pre-commit check, I have to specify which of the two solutions is used as a context (workspace). 

Do I have to write a custom script which decides which solution should be used depending on the staged files, or is there a way to configure this in Husky.Net already?

**Comment by alirezanet** [comment, marked as answer]
Hi @MichalKlinka  
If your solutions are in separate paths you should be able to use[ `include`](https://alirezanet.github.io/Husky.Net/guide/task-configuration.html#configuration) in `task-runner.json` and to use glob pattern to select files ... 
or did I misunderstand the problem?

**Reply by MichalKlinka** [reply]
Thank you for pointing this setting out, I didn't realise it filtered out the staged files.

**Reply by alirezanet** [reply]
No worries, We should improve the documentation 😉
